doc/nrf/installation/install_ncs.rst

```diff
@@ -0,0 +1,607 @@
+
+      If you have received a custom URL for installing the toolchain, you can use the following command to set it as default, replacing the respective parameters:
+
+      .. parsed-literal::
+         :class: highlight
+
+         nrfutil toolchain-manager config --set toolchain-index=*custom_toolchain_URL*
+
+      If you have received a custom bundle ID for installing a specific toolchain version, you can use the following commands to provide it, replacing the respective parameter:
+
+      .. parsed-literal::
+         :class: highlight
+
+         nrfutil toolchain-manager install --bundle-id *custom_bundle_ID*
+
+      To check the current configuration setting, use the ``nrfutil toolchain-manager config --show`` command.
+      To read more about these commands, use the ``nrfutil toolchain-manager --help`` command.
+
+.. _cloning_the_repositories_win:
+.. _cloning_the_repositories:
+
+.. rst-class:: numbered-step
+
+Get the |NCS| code
+******************
+
+Every |NCS| release consists of a combination of :ref:`ncs_git_intro` repositories at different versions and revisions, managed together by :ref:`ncs_west_intro`.
+The revision of each of those repositories is determined by the current revision of the main (or manifest) repository, `sdk-nrf`_.
+Simply put, you can work with the following versions of the |NCS|:
+
+.. list-table::
+   :header-rows: 1
+
+   * - |NCS| version
+     - Required identifier of the revision
+     - Where to find the identifier
+   * - Specific release (recommended)
+     - Release tag (for example, |release_tt|)
+     - :ref:`Release_notes` of the release
+   * - :ref:`Development tag <dm-revisions>`
+     - Development tag (for example, ``v1.9.2-dev1``)
+     - :ref:`Changelog <release_notes>` of the tag
+   * - Branch
+     - Branch name (for example, ``main``)
+     - `sdk-nrf`_ repository
+
+.. note::
+   Unless you are familiar with the :ref:`development process <dev-model>`, you should always work with a specific release of the |NCS|.
+
+For more information about the repository and development model, see the :ref:`dm_code_base` page.
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      To clone the |NCS| code, complete the following steps:
+
+      1. Open the nRF Connect extension in |VSC| by clicking its icon in the :guilabel:`Activity Bar`.
+      #. In the extension's :guilabel:`Welcome View`, click on :guilabel:`Manage SDKs`.
+         The list of actions appears in the |VSC|'s quick pick.
+      #. Click :guilabel:`Install SDK`.
+         The list of available SDK versions appears in the |VSC|'s quick pick.
+      #. Select the SDK version to install.
+         In most cases, you want the SDK version that is either matching or higher than the :ref:`Toolchain version you have chosen <gs_installing_toolchain>`.
```

Comment:
This is not true. Mostly other way around, but not exactly.
Keep in mind that the users first choose the SDK they will work with, not the toolchain, although first they need to install the toolchain.

Something like this:
```suggestion
         Make sure that your active toolchain's _major.minor_ version is at least the same or higher than the SDK version that you intend to install.
```

Reply:
I don't think that's true either, the toolchain will only be built if something changes, so NCS 2.6, 2.7, 2.908 could be released and if the toolchain doesn't change then it will still be toolchain 2.5.x

Reply:
In which case the only suggestion we can give is to install the latest and greatest and if something doesn't work in the future, then upgrade.

Reply:
I will include the recommendation to install the latest released version of the SDK and the toolchain. I think considerations about discrepancies in the versioning should go somewhere else, potentially in Updating tools and repositories (that is, separate PR).